### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by ravindragullapalli** [rubyAce71697/cricket-score-applet#26]
Since a month or so, this app stopped working.
In console, it is showing the below error. error
![screenshot from 2017-07-23 19-56-13](https://user-images.githubusercontent.com/3313079/28500123-fe0cdcc6-6fb2-11e7-9bee-be6d9497beec.png)

**Comment by ravindragullapalli** [comment]
Seems the URL is not working anymore. Another project which loads scores from the RSS. However, it shows the scores in the terminal. Might be useful. Here is the link - https://gist.github.com/arulrajnet/fb71169c35180f9d9abd

**Comment by brispete** [comment]
Looks related to Cricinfo's new website. Shame. This is a lovely thing to have on my desktop when I'm working.

**Comment by ravindragullapalli** [comment]
Yes it is very useful app. I don't know python. Else I also love to contribute.

**Comment by rawcoder** [comment]
On 29 July 2017 9:18:07 PM IST, Ravindra Gullapalli <notifications@github.com> wrote:
>Yes it is very useful app. I don't know python. Else I also love to
>contribute.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/rubyAce71697/cricket-score-applet/issues/26#issuecomment-318839420

Sorry, I was busy with other things.

Seems like they broke the JSON API.  Will try to fix it as soon as I get some time.
-- 
Sent from my mobile device.

**Comment by rawcoder** [comment]
It looks like they've completely re-designed the API.  Will have re-write the extractor from scratch.
I'm not sure if I'd be able to put that much time and effort in it.

I've updated the README to reflect the same.

**Comment by rubyAce71697** [comment]
can you reinstall and try once?

**Comment by ravindragullapalli** [comment]
Hi, I had upgraded my system to Ubuntu 18.04 LTS (Bionic). 

While installing, I am getting the error 

**E: The repository 'http://ppa.launchpad.net/rawcoder/cricket-score-indicator/ubuntu bionic Release' does not have a Release file.**

Tried to install sing pip

Then I got the error

**Could not find a version that satisfies the requirement gi.repository (from cricket_score_indicator) (from versions: )
No matching distribution found for gi.repository (from cricket_score_indicator)**
